java-squid/src/test/java/org/sonar/java/StrutsTest.java

```diff
@@ -89,7 +89,8 @@ public void measures_on_project() throws Exception {
     assertThat(metrics.get("statements").intValue()).isEqualTo(6403);
     assertThat(metrics.get("complexity").intValue()).isEqualTo(3957 - 145 /* SONAR-3793 */ - 1 /* SONAR-3794 */);
     assertThat(metrics.get("comment_lines").intValue()).isEqualTo(7605);
-    assertThat(metrics.get("public_api").intValue()).isEqualTo(1340);
```

Comment:
We should probably have two tests: one with the value of the property set to true and the other one to false, to actually see the difference.